VideoLocker/res/layout/discussion_responses_response_row.xml

```diff
@@ -24,7 +24,7 @@
 
             <org.edx.mobile.view.custom.ETextView
                 android:id="@+id/discussion_responses_comment_body_text_view"
-                android:layout_width="match_parent"
+                android:layout_width="wrap_content"
```

Comment:
Note that if we're assuming that we'll be supporting the RTL language in RTL mode, then `TextView` should be able to take care of it's text direction. If we somehow don't have this assumption, then we can just have the text direction set to `locale` in the global style for `TextView` and similar widgets. We don't need to change all of their layouts to be `wrap_content`.